README.md

```diff
@@ -1,101 +1,107 @@
+# Quick Start Install
 
+Make sure you've installed Node.js - Ghost is fully compatible with **Node v0.10.x**. We recommend **Node 0.10.36** for production use.
 
-## Getting Started
+Ghost is also compatible with **Node v0.12** and **io.js v1.2**, but please note that these versions are more likely to run into installation problems. May contain nuts. Please use the [forum](https://ghost.org/forum/installation/) for help.
 
-There are a few different ways to install Ghost, take care to use the method which best suits your needs.
+1. Download the [latest release](https://ghost.org/download/) of Ghost
+1. Unzip in the location you want to install
+1. Fire up a terminal
+1. `npm install --production`
+1. Start Ghost!
+    - Local environment: `npm start`
+    - On a server: `npm start --production`
+1. `http://localhost:2368/ghost` :tada:
 
-**Please note** - the downloadable zip files we provide on [Ghost.org](http://ghost.org/download), the [GitHub releases page](https://github.com/TryGhost/Ghost/releases), and via npm are pre-built packages designed for getting setup quickly. Cloning from the git repository requires you to install several dependencies and build the assets yourself. 
+More [install docs](http://support.ghost.org/installation/) here in case you got stuck.
 
-### Compatibility
 
-Ghost is fully compatible with **Node v0.10.x**, 0.10.36 is the recommended version of Node for running Ghost in production.
+# Developer Install (from git)
 
-Ghost is also compatible with **Node v0.12** and **io.js-v1.2**. Please note that these versions are more likely to run into installation problems, please use the [support forum](https://ghost.org/forum/installation/) for help with install issues.
+Install Node.js. 
 
-Ghost may also be compatible with other io.js versions, providing binaries are available for sqlite3, and for node-sass if you are building Ghost from source rather than using a pre-built zip file.
+```bash
+# Node v0.10.x - full support
+# Node v0.12.x and io.js v1.2 - partial support
+#
+# Choose wisely
+```
 
+Install grunt. No prizes here.
 
-### Install from zip (fastest & best for bloggers)
+```bash
+npm install -g grunt-cli
+```
 
-If you just want to get a Ghost blog running in the fastest time possible, this method is for you.
+Install Ghost. If you're running locally, use [master](https://github.com/TryGhost/Ghost/tree/master). For production, use [stable](https://github.com/TryGhost/Ghost/tree/stable). :no_entry_sign::rocket::microscope:
 
-For detailed instructions on various platforms, visit the [Ghost Installation Guide](http://support.ghost.org/installation/). If you get stuck, help is available on [our support site](http://support.ghost.org/).
+```bash
+npm install
```

Comment:
You should at least say "clone the repo and then..." kind of like how you
said "install node and then...".  You just went from "install grunt
globally! now install all the npm stuff!" <- missing a step if people
turned off their brains and are just reading along (which is what
instructions are for, after all)

On Thu, Mar 12, 2015 at 2:43 PM, JohnONolan <notifications@github.com>
wrote:

> In README.md
> <https://github.com/TryGhost/Ghost/pull/5029#discussion_r26342524>:
>
> >
> > -For detailed instructions on various platforms, visit the [Ghost Installation Guide](http://support.ghost.org/installation/). If you get stuck, help is available on [our support site](http://support.ghost.org/).
> > +```bash
> > +npm install
>
> That would definitely be clear. My question is: would it be clutter? Is it
> safe to plan that someone following the developer install "(from git)"
> should know how to git?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TryGhost/Ghost/pull/5029/files#r26342524>.
>